syntax = "proto3";

message TableName {
    string name = 1;
}